tests/it/e2e_rpc_server.rs

```diff
@@ -27,6 +27,40 @@ fn e2e_deploy_and_invoke_contract_against_rpc_server() {
         .success();
 }
 
+// e2e tests are ignore by default
+#[test]
+#[ignore]
+fn e2e_install_deploy_and_invoke_contract_against_rpc_server() {
```

Comment:
I don't understand how to run this test and given that it's late Friday I didn't want to bother anyone asking. I hope this works, but let's figure this out on Monday.